betterproto/templates/template.py

```diff
@@ -77,21 +94,42 @@ async def {{ method.py_name }}(self{% if method.input_message and method.input_m
         request.{{ field.name }} = {{ field.name }}
             {% endif %}
         {% endfor %}
+        {% endif %}
 
         {% if method.server_streaming %}
+            {% if method.client_streaming %}
```

Comment:
some indentation is off around here

Reply:
I'll take a look at that.  I'm not entirely sure how one likes to indent jinja templates, I can skip the indent of the nested if-statement.

Reply:
Hopefully the new version looks better.